ide/vs-code/README.md

```diff
@@ -35,26 +35,65 @@ module.exports = {
 };
 ```
 
-#### Create Prettier Config
+### Create Prettier Config
 
 - Add `"prettier/prettier": "error",` to the rules in the [.eslintrc.js](/.eslintrc.js)
 - If a local set of prettier rules is needed
-  - in root of project create a file named `.prettierrc` for prettier rules
+  - In root of project create a file named `.prettierrc` for prettier rules
   - [Prettier rules can be found here](https://prettier.io/docs/en/options.html)
 
-#### Custom Configurations
+### Custom Configurations
 
 Additional linting rules can be found at <https://eslint.org/docs/rules/>
 
 - Suggested CSS linting rules [can be found here](/css/.csslintrc).
-- A few suggested javascript [configuration objects can be found here](/javascript/.eslintrc)
-- A few additional javascript [linting rules can be found here](/javascript/.jscsrc)
+- A few suggested JavaScript [configuration objects can be found here](/javascript/.eslintrc)
+- A few additional JavaScript [linting rules can be found here](/javascript/.jscsrc)
+
+## Pairing with [Live Share](https://visualstudio.microsoft.com/services/live-share/)
+
+### Install the Live Share Extension
+
+- Search for **"Live Share"** in the Extensions Marketplace, published by Microsoft
+
+![Install the Live Share extension](./images/live-share-install.png)
+
+### Start or Join a Session
+
+- Select the now-installed Live Share extension on the left icon bar
+- Sign in through GitHub when/if prompted
```

Comment:
```suggestion
- Click "Share Now" button, and sign in through GitHub when/if prompted
```

Reply:
I like your wording a lot better. I'm pretty sure the first time that I used Live Share I joined Drew's session, rather than starting my own, and I'm 99% sure I was still prompted to sign in through GitHub. It's hard to verify this because I haven't been prompted to sign in again since.

Maybe I'll expand on your wording a little bit to include both potential actions, and we can see what another person's experience is.